app/src/androidTest/java/com/duckduckgo/app/email/ui/EmailProtectionSignOutViewModelTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.email.ui
+
+import app.cash.turbine.test
```

Comment:
really nice to see how this works.

Reply:
Love to see this also coming here @marcosholgado